crates/store/re_datafusion/src/dataframe_query_common.rs

```diff
@@ -190,6 +191,49 @@ impl DataframeQueryTableProvider {
             chunk_request,
         })
     }
+
+    fn column_to_selector(column: &Column) -> Option<ComponentColumnSelector> {
+        ComponentColumnSelector::from_str(column.name()).ok()
+    }
+
+    fn compute_column_is_not_null_filter(
```

Comment:
This could use a docstring to explain that what this checks for, especially since the function name can be read two ways (does it return true for the filter `foo == 42` since that is not a null filter?)